test_plus/test.py

```diff
@@ -381,6 +383,28 @@ def __init__(self, *args, **kwargs):
         self.client_class = get_api_client()
         super(APITestCase, self).__init__(*args, **kwargs)
 
+    # APITest requests now default to JSON
+    def request(self, method, url_name, *args, **kwargs):
+        data = kwargs.get("data")
+        kwargs["data"] = json.dumps(data) if data is not None else None
+        kwargs["extra"] = {
+            "content_type": "application/json",
+            "HTTP_X_REQUESTED_WITH": "XMLHttpRequest",
```

Comment:
🤔 I'm not sure if we need this.

Reply:
The `kwargs["extra"] ` part or the `kwargs["data"] ` as well?

The `kwargs["extra"]` part is what I'm most unclear on, but I thought the `kwargs["data"]` part was the MAIN point: making the APITestCase requests _explicitly_ JSON.

Or is that something DRF is just already doing for us now?

Reply:
The #L392 `"HTTP_X_REQUESTED_WITH": "XMLHttpRequest",` part. The content_type is required, but I am not sure why we'd need the `HTTP_X_REQUESTED_WITH` part. Did someone request it or am I missing something and it's needed?